src/System.Runtime.Numerics/src/System/Numerics/BigCalc.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System.Numerics
+{
```

Comment:
Yes, seems to be left-over or at least unused. You can probably remove the "partial" declaration unless you had plans to actually use it for something.